### ISSUE TITLE
remove various warnings issued by Spawn

### ISSUE THREAD

**Issue by mwetter** [lbl-srg/modelica-buildings#2220]
Simulating `Buildings.ThermalZones.EnergyPlus.Validation.Schedule.EquipmentSchedule` in Dymola leads to the warnings below. 
This issue is to remove them.

The output is from 753a530ff927d777694a1fa09c9f0b6636dd5b8b (the current master).

```
In EquipmentSchedule.building: EnergyPlus Warning: ManageSizing: For a zone sizing run, there must be at least 1 Sizing:Zone input object. SimulationControl Zone Sizing option ignored.
In EquipmentSchedule.building: EnergyPlus Warning: ManageSizing: For a plant sizing run, there must be at least 1 Sizing:Plant object input. SimulationControl Plant Sizing option ignored.
In EquipmentSchedule.building: EnergyPlus Warning: ProcessScheduleInput: Schedule:Constant="SPAWN-RADIANTGAINS-SCHEDULE", Blank Schedule Type Limits Name input -- will not be validated.
Model: Buildings.ThermalZones.EnergyPlus.Validation.Schedule.EquipmentSchedule
Integration started at 0 using integration method:
cvode from sundials

In EquipmentSchedule.building: EnergyPlus Warning: Calculated Relative Humidity out of range (PsyRhFnTdbWPb) 
In EquipmentSchedule.building: EnergyPlus Warning:  Routine=Unknown, Environment=SPAWN-RUNPERIOD, at Simulation time=01/01 00:50 - 00:50
In EquipmentSchedule.building: EnergyPlus Warning:  Dry-Bulb= -7.72 Humidity Ratio= 2.083E-003 Calculated Relative Humidity [%]= 104.60
In EquipmentSchedule.building: EnergyPlus Warning: Relative Humidity being reset to 100.0%
```

**Comment by mwetter** [comment]
@kbenne : With Spawn-0.1.0-3ec0a1fa6e-Linux.tar.gz, the warning about the humidity is still there:
```
Model: Buildings.ThermalZones.EnergyPlus.Examples.SingleFamilyHouse.EquipmentSchedule
Integration started at 0 using integration method:
cvode from sundials

3600.000 EquipmentSchedule.building: Warning from EnergyPlus: Calculated Relative Humidity out of range (PsyRhFnTdbWPb) 
3600.000 EquipmentSchedule.building: Warning from EnergyPlus:  Routine=Unknown, Environment=SPAWN-RUNPERIOD, at Simulation time=01/01 00:50 - 00:50
3600.000 EquipmentSchedule.building: Warning from EnergyPlus:  Dry-Bulb= -7.89 Humidity Ratio= 2.083E-003 Calculated Relative Humidity [%]= 106.16
3600.000 EquipmentSchedule.building: Warning from EnergyPlus: Relative Humidity being reset to 100.0%
Integration terminated successfully at T = 172800
   CPU-time for integration                  : 0.623389 seconds
   CPU-time for initialization               : 1.33361 seconds
   Number of result points                   : 1075
   Number of grid points                     : 501
   Number of accepted steps                  : 2236
   Number of rejected steps                  : 242
   Number of f-evaluations (dynamics)        : 4643
   Number of non-linear iteration            : 3497
   Number of non-linear convergence failures : 0
   Number of Jacobian-evaluations            : 288
   Number of crossing function evaluations   : 2735
   Number of model time events               : 288
   Number of state events                    : 0
   Number of step events                     : 0
172800.000 EquipmentSchedule.building: Warning from EnergyPlus:  ** Warning ** Calculated Relative Humidity out of range (PsyRhFnTdbWPb)
172800.000 EquipmentSchedule.building: Warning from EnergyPlus: 
```
Is this checked and passes on your CI test?

**Comment by kbenne** [comment]
The model I was testing didn't show that one. I think it was the small office reference building that I was using in Chicago. Anyway, I'll see what I can do. This one might not be addressed by release. I do not have automated tests to capture the warning count so CI is blissfully unaware.

**Comment by mwetter** [comment]
This will be corrected after Buildings 8.0.0 release.

**Comment by kbenne** [comment]
@mwetter I've fixed two reoccurring warnings. 

1. Invalid people activity value
2. Out of range humidity values

I've been specifically using the Buildings.ThermalZones.EnergyPlus.Examples.SingleFamilyHouse.AirHeating model as a test case, but if you know of other models that are generating different warnings the let me know. I think this clears up the common warnings that we are seeing.

Note that item 2 is significant beyond simply fixing a warning. The warning was a symptom of the underlying problem that I was not updating the moisture balance correctly.
